api/README.md

```diff
@@ -122,16 +134,67 @@ try {
   console.error(`${error.name}: ${error.message}`);
 }
 ```
+</details>
 
-#### Check that the address belongs to some program
+
+### Upload code
+Use `api.code.upload` method to create `upload_code` extrinsic
+
+<details>
+<summary>Example</summary>
 
 ```javascript
-const programId = '0x0000000000000000000000000000000000000000000000000000000000000000';
-const programExists = await api.program.exists(programId);
-console.log(`Program with address ${programId} ${programExists ? 'exists' : "doesn't exist"}`);
+const code = fs.readFileSync('path/to/program.opt.wasm');
+const { codeHash } = gearApi.code.upload(code);
+gearApi.code.signAndSend(alice, () => {
+  events.forEach(({ event: { method, data } }) => {
+    if (method === 'ExtrinsicFailed') {
+      throw new Error(data.toString());
+    } else if (method === 'CodeSaved') {
+      console.log(data.toHuman());
+    }
+  });
+});
+```
+</details>
+
+### Create program from uploaded code on chain
+Use `api.program.create` method to create `create_program` extrinsic
+<details>
+<summary>Code example</summary>
+
+```javascript
+const codeId = '0x...'
+
+const program = {
+  codeId,
+  gasLimit: 1000000,
+  value: 1000,
+  initPayload: somePayload,
+};
+
+try {
+  const { programId, salt, extrinsic } = gearApi.program.create(program, meta);
+} catch (error) {
+  console.error(`${error.name}: ${error.message}`);
+}
```

Comment:
Do you think that we should mention `try-catch`?
Imo it's up to user and hurts readability.